compiler/rustc_interface/src/tests.rs

```diff
@@ -738,6 +738,7 @@ fn test_unstable_options_tracking_hash() {
     untracked!(unstable_options, true);
     untracked!(validate_mir, true);
     untracked!(verbose, true);
+    untracked!(write_long_types_to_disk, true);
```

Comment:
This needs to be set to a value that's *not* default

```suggestion
    untracked!(write_long_types_to_disk, false);
```